test/specs/new/escape_within_emphasis.md

```diff
@@ -0,0 +1,5 @@
+**strong text\[**\]
+
+**strong text\\\[**\]
+
+_em\[pha\]\(sis\)_
```

Comment:
You have one and three backslashes; could you add an example with 2 backslashes just for completeness?

Reply:
What about `**strong text\\[**\]`

Reply:
That's one of the cases that is not yet handled correctly as of this PR.